### ISSUE TITLE
Intermediate Checkpoints

### ISSUE THREAD

**Issue by HarmanDotpy** [mlfoundations/open_clip#538]
Hi, thanks for the great repo!

I was wondering if there is a possibility to get intermediate checkpoints  (for the n(=32) epochs) for some common clip models. for eg. (any/some of) CLIP-VIT-B32 pre-trained on laion/yfcc/cc?

Thanks!

**Comment by mehdidc** [comment]
Hi @HarmanDotpy , you can find intermediate checkpoints at https://huggingface.co/laion/scaling-laws-openclip/tree/main/full_checkpoints, from our openclip scaling laws paper (https://github.com/LAION-AI/scaling-laws-openclip). For instance, for ViT B-32  trained on 32 epochs of LAION-400M (63.2% zero-shot ImageNet-1k accuracy), it's available at https://huggingface.co/laion/scaling-laws-openclip/tree/main/full_checkpoints/Model-B-32_Data-400M_Samples-13B_lr-1e-3_bs-86k. For more info about each model, each folder name corresponds to a row in https://github.com/LAION-AI/scaling-laws-openclip/blob/master/trained_models_info.csv
